test/framework/result_gtest_checkers.hpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_RESULT_GTEST_CHECKERS_HPP
+#define IROHA_RESULT_GTEST_CHECKERS_HPP
+
+#include "common/result.hpp"
+
+#include <gtest/gtest.h>
+
+namespace framework {
+  namespace expected {
+    namespace detail {
+      std::string getMessage(const std::string &s) {
+        return s;
+      }
+
+      template <typename T>
+      auto getMessage(const T &o) -> std::enable_if_t<
+          std::is_same<decltype(o.toString()), std::string>::value,
+          std::string> {
+        return o.toString();
+      }
+
+      template <typename T>
+      auto getMessage(const T &o) -> std::enable_if_t<
+          std::is_same<decltype(o->toString()), std::string>::value,
+          std::string> {
+        return o->toString();
+      }
+    }  // namespace detail
+
+    template <typename V, typename E>
+    void expectResultValue(const iroha::expected::Result<V, E> &r) {
+      EXPECT_TRUE(iroha::expected::hasValue(r))
+          << "Value expected, but got error :"
```

Comment:
Suggest to put colon before space, not after (here and below as well).

Reply:
sure, that's a typo, thanks :eyes: